src/pages/Recommender.tsx

```diff
@@ -95,18 +97,19 @@ class Recommender extends React.Component<RouteComponentProps, IRecommenderState
     renderAnswerOptions() : ReactNode {
         const elems:React.ReactElement[] = [];
 
+        let key = 0
         for (let answer of this.answer_options) {
             let stringvalue = getAnswerString(answer);
             elems.push(
-                <button onClick={() => this.onAnswerGiven(answer)} className={'btn btn-primary next-btn edu-btn div-spacing'}>
+                <button id={Answer_Enum[answer].toLowerCase() + "_btn"} key={key++} onClick={() => this.onAnswerGiven(answer)} className={'btn btn-primary next-btn edu-btn div-spacing'}>
```

Comment:
Hvorfor det her?

Reply:
For at fange elementet via html id i tests

Reply:
For at kunne referer til knapperne i test skal de have en id. `Answer_Enum[answer].toLowerCase() + "_btn"` er bare logik til at genererer svar værdierne i formen: yes_btn, probably_btn osv.